Peer and swarm definitions (#1153)
ipfs/ipfs-docs: docs/concepts/glossary.md

### Swarm

> The Swarm is a term for the network of IPFS peers with which your local node has connections. Swarm addresses are addresses that your local node will listen on for connections from other IPFS peers.

Reviewer: ```suggestion
_Swarm_ is a term for the network of IPFS peers with which your local node has connections. Swarm addresses are addresses that your local node will listen on for connections from other IPFS peers.
```

### Switch

> In [libp2p](#libp2p), the switch is the component responsible for composing multiple [transports](#transport) into a single interface, allowing application code to [dial](#dialing) peers without having to specify which transport to use.

Reviewer: ```suggestion
In [libp2p](#libp2p), a _switch_ is a component responsible for composing multiple [transports](#transport) into a single interface, allowing application code to [dial](#dialing) peers without having to specify which transport to use.
```

> The switch also coordinates the "connection upgrade" process, which promotes a "raw" connection from the transport layer into one that supports [protocol negotiation](https://docs.libp2p.io/concepts/protocols/#protocol-negotiation), [stream multiplexing](../concepts/libp2p/#stream-multiplexing), and secure communications.

Reviewer: ```suggestion
Switches also coordinate the _connection upgrade_ process, which promotes a _raw_ connection from the transport layer into one that supports [protocol negotiation](https://docs.libp2p.io/concepts/protocols/#protocol-negotiation), [stream multiplexing](../concepts/libp2p/#stream-multiplexing), and secure communications.
```

> Sometimes called ["Swarm"](#swarm) for historical reasons.

Reviewer: ```suggestion
Sometimes called [swarm](#swarm) for historical reasons.
```
